src/hooks/useLongPress.ts

```diff
@@ -85,11 +86,14 @@ const useLongPress = (
     [pressed],
   )
 
-  const onContextMenu = useCallback((e: React.MouseEvent) => {
-    e.preventDefault()
-    e.stopPropagation()
-    selection.clear()
-    dispatch(editing({ value: false }))
+  const onContextMenu = useCallback((e: React.PointerEvent) => {
+    // Double tap activation of context menu produces a pointerType of `touch` whereas long press activation of context menu produces pointer type of `mouse`
+    if (!isTouch || (isTouch && e.nativeEvent.pointerType === 'touch')) {
```

Comment:
You can remove `isTouch &&` since it will always be true.

Reply:
I see. I forgot about the short-circuiting.